src/textinput/example/index.ts

```diff
@@ -39,8 +39,9 @@ export class App extends WidgetBase<WidgetProperties> {
 					type: 'text',
 					placeholder: 'Hello, World',
 					value: this._value1,
-					onChange: (event: TypedTargetEvent<HTMLInputElement>) => {
-						this._value1 = event.target.value;
+					onChange: (event: Event) => {
+						const target = event.target as HTMLInputElement;
+						this._value1 = target.value;
```

Comment:
any reason to define `target` here instead of the `this._value1 = (event.target as HTMLInputElement).value` approach used elsewhere

Reply:
I'll make that change